src/components/PaymentCard/PaymentCard.js

```diff
@@ -3,88 +3,126 @@ import PropTypes from 'prop-types';
-  const LogoComponent =
-    paymentMethodSpecificParams &&
-    CardTypesIcons[paymentMethodSpecificParams.variant]
-      ? CardTypesIcons[paymentMethodSpecificParams.variant]
-      : CardTypesIcons[details.paymentMethod] || null;
+
+  let LogoComponent = null;
+  let methodTitle = '';
+  if (
+    paymentMethod === 'card' &&
+    CardTypes[paymentMethodSpecificParams.variant]
+  ) {
+    LogoComponent = CardTypes[paymentMethodSpecificParams.variant]?.icon;
+    methodTitle = CardTypes[paymentMethodSpecificParams.variant]?.title;
+  } else if (paymentMethod === 'paypal') {
+    LogoComponent = CardTypes[paymentMethod]?.icon;
+    methodTitle = CardTypes[paymentMethod]?.title;
+  } else {
+    LogoComponent = CardTypes[details.paymentMethod]?.icon;
+    methodTitle = CardTypes[details.paymentMethod]?.title;
+  }
 
   return (
-    <CardWrapStyled type={details.paymentMethod}>
+    <Card withBorder type={details.paymentMethod}>
       {isDataLoaded ? (
         <CardStyled>
-          {LogoComponent && (
-            <CardTypeStyled>
-              <LogoComponent />
-            </CardTypeStyled>
-          )}
-          {paymentMethodSpecificParams?.lastCardFourDigits && (
-            <CardNumberStyled>
-              **** **** **** {paymentMethodSpecificParams.lastCardFourDigits}
-            </CardNumberStyled>
-          )}
-          {INAPPS.includes(paymentMethod) && (
-            <CardNumberStyled>
-              {t('Billing via')}{' '}
-              <MethodNameStyled>{paymentMethod}</MethodNameStyled>
-            </CardNumberStyled>
-          )}
-          {paymentMethodSpecificParams?.cardExpirationDate && (
-            <CardExpirationStyled>
-              <CardExpirationLabel>{t('Expiry date')}</CardExpirationLabel>
-              <CardExpirationDateStyled>
-                {paymentMethodSpecificParams.cardExpirationDate}
-              </CardExpirationDateStyled>
-            </CardExpirationStyled>
-          )}
-          {paymentMethodSpecificParams?.holderName &&
-            !paymentMethodSpecificParams?.cardExpirationDate && (
-              <CardExpirationStyled>
-                <CardExpirationLabel>{t('Holder name')}</CardExpirationLabel>
-                <CardExpirationDateStyled>
-                  {paymentMethodSpecificParams.holderName}
-                </CardExpirationDateStyled>
-              </CardExpirationStyled>
-            )}
-          <CardEditStyled
-            onClick={() => {
-              showInnerPopup({
-                type: POPUP_TYPES.paymentDetails,
-                data: details
-              });
-              window.dispatchEvent(
-                new CustomEvent('MSSDK:edit-payment-button-clicked', {
-                  detail: {
-                    paymentMethod
-                  }
-                })
-              );
-            }}
-          >
-            {t('Edit payment info')}
-          </CardEditStyled>
+          <CardHeaderStyled>Payment method</CardHeaderStyled>
```

Comment:
I would ask designers once again if we need this duplication of header 'Payment method'

Reply:
I've asked about that and also about that holder's name for Paypal